application/models/Question.php

```diff
@@ -1002,7 +1002,7 @@ public function search()
         $criteria2->compare($qid_reference, $this->title, true, 'OR');
 
         if ($this->gid != '') {
-            $criteria->compare('groups.gid', $this->gid, true, 'AND');
+            $criteria->compare('groups.gid', $this->gid, false, 'AND');
```

Comment:
See https://www.yiiframework.com/doc/api/1.1/CDbCriteria#compare-detail this change the partial match then add 1 and 10 and 11 and … if `$this->gid` is 1, `false` fixed it

Question : why it's an issue in pgsql ?